.github/actions/e2e/install-prometheus/action.yaml

```diff
@@ -42,4 +42,12 @@ runs:
           -f ./.github/actions/e2e/install-prometheus/values.yaml \
           --set prometheus.prometheusSpec.remoteWrite[0].url=https://aps-workspaces.${{ inputs.region }}.amazonaws.com/workspaces/${{ inputs.workspace_id }}/api/v1/remote_write \
           --set prometheus.prometheusSpec.remoteWrite[0].sigv4.region=${{ inputs.region }} \
-          --set prometheus.serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::${{ inputs.account_id }}:role/prometheus-irsa-${{ inputs.cluster_name }}"
\ No newline at end of file
+          --set prometheus.serviceAccount.annotations."eks\.amazonaws\.com/role-arn"="arn:aws:iam::${{ inputs.account_id }}:role/prometheus-irsa-${{ inputs.cluster_name }}" \
```

Comment:
Have we thought about using something like `envsubst` with our values? I think we are getting to the point where the number of values that we are setting on the command-line here is getting a little much and it might be nice to just set everything in a `values.yaml` file that we can override with environment variables

Reply:
Definitely a good idea, let's try and do this if we add any more here.